### DISCUSSION TITLE
ruckus/ZD1200LicensesAndSupport

### DISCUSSION THREAD

**Discussion by giscus** [ms264556/Hackery#14] [Comments]
# ruckus/ZD1200LicensesAndSupport

Hackery

https://ms264556.net/ruckus/ZD1200LicensesAndSupport

**Comment by smfilip** [comment]
And where exactly are files for ZD1200?

**Comment by ms264556** [comment]
Ruckus let me know that although the ZD1200 isn't sold anymore they still sell support and AP licenses. And a reseller advised that these licenses are purchasable now even for used/gray-market devices.

I definitely don't want to ruin anyone's business. I'm just trying to rescue abandoned devices from becoming e-waste. So I took the information down.

**Comment by rhy7s** [comment]
I'm not clear on the situation with releases after firmware signing was introduced. After downgrading and applying the patch, can we then upgrade to the latest release and apply the patch again? Or we need to stick to a release prior to 10.5.1.0.255?

**Reply by ms264556** [reply]
Ruckus advised that they will still happily sell anyone licenses and support for any ZD1200, which is the only device which can run 10.5.1. Since my aim is to prevent e-waste, rather than hurt anyone's business, I don't share a ZD1200 patch anymore.

Contact me privately [here](https://forums.servethehome.com/index.php?members/ms264556.37738/) or [here](https://www.reddit.com/user/Famous-Fishing-1554/) if you have questions about a previously-applied ZD1200 patch & I'll help you out.

ZD AP licenses are persistent through upgrades and factory resets, so if you patched extra AP licenses onto your ZoneDirector then you can upgrade to a later/earlier release & you will still have the extra AP licenses.
